batch/src/main/resources/es_index.json

```diff
@@ -0,0 +1,349 @@
+            },
+            "entryDate": {
+              "type": "date",
+              "format": "strict_date_optional_time||epoch_millis",
+              "index": "not_analyzed"
+            },
+            "lastUpdateDate": {
+              "type": "date",
+              "format": "strict_date_optional_time||epoch_millis",
+              "index": "not_analyzed"
+            },
+            "organizationName": {
+              "type": "string",
+              "analyzer": "welsh_split_analyzer"
+            },
+            "poBoxNumber": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "postTown": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "postcode": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "postcodeType": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "proOrder": {
+              "type": "long",
+              "index": "not_analyzed"
+            },
+            "processDate": {
+              "type": "date",
+              "format": "strict_date_optional_time||epoch_millis",
+              "index": "not_analyzed"
+            },
+            "recordIdentifier": {
+              "type": "byte",
+              "index": "not_analyzed"
+            },
+            "startDate": {
+              "type": "date",
+              "format": "strict_date_optional_time||epoch_millis",
+              "index": "not_analyzed"
+            },
+            "subBuildingName": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "thoroughfare": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "udprn": {
+              "type": "integer",
+              "index": "not_analyzed"
+            },
+            "uprn": {
+              "type": "long",
+              "index": "not_analyzed"
+            },
+            "welshDependentLocality": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "welshDependentThoroughfare": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "welshDoubleDependentLocality": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "welshPostTown": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "welshThoroughfare": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "pafAll": {
+              "type": "string",
+              "analyzer": "welsh_split_analyzer"
+            }
+          }
+        },
+        "uprn": {
+          "type": "long",
```

Comment:
It should be the same, yes.
It's easier to have it there, we group pafs and nags by uprn that results in a hybrid address.
Removing the `uprn` column will add some code (and complexity), it's a small duplication, I think it does not hurt.